src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -47,7 +47,7 @@
 #include "LuaEngine.h"
 #endif
 
-constexpr uint32 DEF_CANNOT_REACH = 5 * IN_MILLISECONDS; // this is when creatures are in los / no path to the target, 5s wait time then they return to spawn pos with evade
+constexpr uint32 DEF_CANNOT_REACH = 7 * IN_MILLISECONDS; // this is when creatures are in los / no path to the target, 5s wait time then they return to spawn pos with evade
```

Comment:
the comment should be updated too, also what about making this value configurable via `worldserver.conf` ? so we can easily test different values without recompiling..

Reply:
Good idea! Will add it tomorrow.